### PR TITLE
Bug: app auth cache keys missing app_id

### PR DESCRIPTION
**Pull request by zegl**
This makes the cache multi-app aware, and so that it can handle JWT tokens from multiple apps

### CODE REVIEW

File: githubkit/auth/app.py [hunk 1]
```diff
@@ -74,15 +74,17 @@ def _create_jwt(self) -> str:
         )
 
     def get_jwt(self) -> str:
-        if not (token := self.cache.get(self.JWT_CACHE_KEY)):
+        cache_key = self.JWT_CACHE_KEY.format("app_id", self.app_id)
```
Comment (yanyongyu) [inline githubkit/auth/app.py:77]:
```suggestion
        cache_key = self.JWT_CACHE_KEY.format(app_id=self.app_id)
```

File: githubkit/auth/app.py [hunk 2]
```diff
@@ -74,15 +74,17 @@ def _create_jwt(self) -> str:
...
+        if not (token := self.cache.get(cache_key)):
             token = self._create_jwt()
-            self.cache.set(self.JWT_CACHE_KEY, token, timedelta(minutes=8))
+            self.cache.set(cache_key, token, timedelta(minutes=8))
         return token
 
     async def aget_jwt(self) -> str:
-        if not (token := await self.cache.aget(self.JWT_CACHE_KEY)):
+        cache_key = self.JWT_CACHE_KEY.format("app_id", self.app_id)
```
Comment (yanyongyu) [inline githubkit/auth/app.py:84]:
```suggestion
        cache_key = self.JWT_CACHE_KEY.format(app_id=self.app_id)
```

File: githubkit/auth/app.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ class AppAuth(httpx.Auth):
     permissions: Union[Unset, "AppPermissionsType"] = UNSET
     cache: "BaseCache" = DEFAULT_CACHE
 
-    JWT_CACHE_KEY = "githubkit:auth:app:jwt"
+    JWT_CACHE_KEY = "githubkit:auth:app:jwt:{app_id}"
```
Comment (yanyongyu) [inline githubkit/auth/app.py:42]:
```suggestion
    JWT_CACHE_KEY = "githubkit:auth:app:{app_id}:jwt"
```

**Review by yanyongyu — COMMENTED** [review]
It seems the installation cache key also missing the app id.

**Comment by zegl** [comment]
> It seems the installation cache key also missing the app id.

Yeah, but I don't think it's as critical, as different installation IDs imply different app IDs.

**Review by yanyongyu — APPROVED** [review]

**Pull request merged**